### PR TITLE
sync with upstream

### PR DESCRIPTION
**Pull request by gtema**

### CODE REVIEW

File: otcextensions/tests/functional/sdk/volume_backup/v2/test_backup_policy.py [hunk 1]
```diff
@@ -34,9 +34,8 @@ class TestBackupPolicy(TestVbs):
     policy = None
     volume = None
 
-    @classmethod
-    def setUpClass(cls):
-        super(TestBackupPolicy, cls).setUpClass()
+    def setUp(cls):
```
Comment (vladimirhasko) [inline otcextensions/tests/functional/sdk/volume_backup/v2/test_backup_policy.py]: everywhere else you have removed cls and replaced by self. What about here?

**Review by vladimirhasko — COMMENTED** [review]
please have look to my comment

File: otcextensions/tests/functional/sdk/volume_backup/v2/test_backup_policy.py (hunk 1 above)
Comment (gtema) [reply otcextensions/tests/functional/sdk/volume_backup/v2/test_backup_policy.py]: right, not a bug though, but fixed

File: otcextensions/tests/functional/sdk/dms/v1/test_message.py [hunk 2]
```diff
@@ -26,78 +26,76 @@ class TestMessage(base.BaseFunctionalTest):
...
+            self.queue = self.conn.dms.create_queue(
                 name=TestMessage.QUEUE_ALIAS
             )
 
         except openstack.exceptions.BadRequestException:
-            cls.queue = cls.conn.dms.get_queue(TestMessage.QUEUE_ALIAS)
+            self.queue = self.conn.dms.get_queue(TestMessage.QUEUE_ALIAS)
 
-        cls.queues.append(cls.queue)
+        self.queues.append(self.queue)
```
Comment (vineet-pruthi) [inline otcextensions/tests/functional/sdk/dms/v1/test_message.py:40]: shouldn't this be TestMessage.queues.append ?

Comment (gtema) [reply otcextensions/tests/functional/sdk/dms/v1/test_message.py:40]: Nope, we want to add to object queue, and not class queue

Comment (vineet-pruthi) [reply otcextensions/tests/functional/sdk/dms/v1/test_message.py:40]: Ignore the comment :D

**Review by vineet-pruthi — APPROVED** [review]

File: otcextensions/tests/functional/sdk/dms/v1/test_message.py (hunk 2 above)
Comment (vineet-pruthi) [reply otcextensions/tests/functional/sdk/dms/v1/test_message.py:40]: > 
> 
> Nope, we want to add to object queue, and not class queue

Sorry i didnt see your feedback on my comment and then wrote another comment, but thank you for clearing 👍

**Review by vladimirhasko — APPROVED** [review]

**Pull request merged**